src/features/hotspots/settings/HotspotSettings.tsx

```diff
@@ -41,6 +41,10 @@ import { useHotspotSettingsContext } from './HotspotSettingsProvider'
 import Box from '../../../components/Box'
 import BackButton from '../../../components/BackButton'
 import animateTransition from '../../../utils/animateTransition'
+import BluetoothIcon from '../../../assets/images/bluetooth_icon.svg'
+import TransferIcon from '../../../assets/images/transfer_icon.svg'
+// import DiscoveryModeIcon from '../../../assets/images/discovery_mode_icon.svg'
```

Comment:
remove?

Reply:
I kept them for when we added those other code blocks back in